ember/app/ui/cases/detail/circulation/controller.js

```diff
@@ -167,7 +167,13 @@ export default class CasesDetailCirculationController extends Controller {
       variables: {
         id: this.inviteToCirculationWorkItem.id,
         context: JSON.stringify({
-          assign_users: this.selectedIdentities.mapBy("idpId"),
+          assign_users: this.selectedIdentities.map((identity) => {
+            return {
+              idpId: identity.idpId,
+              name: identity.fullName,
```

Comment:
The code in `next_deployment.md` uses `{firstname} {lastname}`, while `identity.fullName` uses `{lastname} {firstname}`, right?

Reply:
fixed the next_deployment.md order of the attributes